msal/src/Microsoft.Identity.Client/Platforms/netstandard1.3/UIParent.cs

```diff
@@ -59,5 +51,24 @@ public UIParent()
         {
             CoreUIParent = new CoreUIParent();
         }
+
+        /// <summary>
+        /// Platform agnostic constructor that allows building an UIParent from a NetStandard assembly. 
+        /// </summary>
+        /// <remarks>Interactive auth is not currently implemented in .net core</remarks>
+        /// <param name="parent">An owner window to which to attach the webview to.
+        /// On Android, it is mandatory to pass in an Activity.
+        /// On all other platforms, it is not required.
+        /// On .net desktop, it is optional - you can either pass a System.Windows.Forms.IWin32Window or an System.IntPtr
+        /// to a window handle or null. This is used to center the webview. </param>
+        /// <param name="useEmbeddedWebview">Flag to determine between embedded vs system browser. Currently affects only iOS and Android. See https://aka.ms/msal-net-uses-web-browser </param>
+        public UIParent(object parent, bool useEmbeddedWebview)
+        {
+            // for the rare case when an application actually uses the netstandard implementation
+            // i.e. other frameworks - e.g. Xamarin.MAC - or MSAL.netstandard loaded via reflection
+            throw new PlatformNotSupportedException("Interactive Authentication flows are not supported when the NetStandard assembly is at runtime. " +
```

Comment:
NetStandard assembly will not be used at runtime in 99% of the cases. But in the rare cases where it will, better to throw a good exception. 

![image](https://user-images.githubusercontent.com/12273384/49018736-1b40aa80-f184-11e8-93d1-d296d0a7ab7c.png)